src/algebra/module/prod.lean

```diff
@@ -28,6 +28,11 @@ instance mul_action_with_zero [monoid_with_zero R] [has_zero M] [has_zero N]
   zero_smul := λ ⟨m, n⟩, prod.ext (zero_smul _ _) (zero_smul _ _),
   ..prod.mul_action }
 
+instance distrib_mul_action_with_zero [semiring R] [add_monoid M] [add_monoid N]
```

Comment:
Does this need `semiring R` and not just `monoid_with_zero R`?

Reply:
This occurs in lots of places

Reply:
Whoops, just `monoid_with_zero R`